Makefile

```diff
@@ -162,7 +162,13 @@ all-tests: eunit test
 docker:
 	@docker build . -t aeternity/epoch:local
 
-smoke-test: system-test-deps docker
+docker-clean:
+	@docker image prune -f
+	@docker container prune -f
```

Comment:
This removes stopped containers.

Reply:
This might be too much? The target has to be run manually though...